### DISCUSSION TITLE
Show native Android media controls when playing media

### DISCUSSION THREAD

**Discussion by bgmulinari** [CommunityToolkit/Maui#1379] [Q&A] [answered]
I just started a project using the MediaElement from the Community Toolkit and it's been great so far, however, I have a doubt regarding the media controls.

My project is an Android app that plays some music files that I embed with it. I was wondering if it's possible to show the native [Android media controls](https://developer.android.com/media/implement/surfaces/mobile) in the notification center/lock screen when a music is playing in my app, much like Spotify does it.

![image](https://developer.android.com/static/images/media/samples/media-controls-android-13.svg)

Is this currently possible with the `MediaElement` from the Community Toolkit?

**Comment by pictos** [comment, marked as answer]
@bgmulinari yes, it's possible, but you will need to implement it by yourself. There's a java code on docs, you just need to translate then to C# and your needs (:

**Reply by bgmulinari** [reply]
Thank you, I'll look that up.

**Comment by CliffAgius** [comment]
Closed as answered
